addons/account/security/ir.model.access.csv

```diff
@@ -60,7 +60,7 @@ access_account_account_invoice,account.account invoice,model_account_account,acc
 access_account_analytic_accountant,account.analytic.account accountant,analytic.model_account_analytic_account,account.group_account_user,1,1,1,1
 access_account_account_type_invoice,account.account.type invoice,model_account_account_type,account.group_account_invoice,1,0,0,0
 access_account_account_tax,account.account.tag,model_account_account_tag,account.group_account_user,1,1,1,1
-access_account_account_tax_user,account.account.tag,model_account_account_tag,account.group_account_invoice,1,0,0,0
+access_account_account_tax_user,account.account.tag,model_account_account_tag,base.group_user,1,0,0,0
```

Comment:
it seems to me adding a 1000 access to model_account_account_tag for purchase_user in the purchase module would be cleaner

also, we have a `test_access_rights.py` file in purchase now, can you take care of adding a test case there?

Reply:
I put it account because I think that will be useful for others to extend. Let's think when you/your team would develop another app that also allows users of the app to create invoices. In that situation, you would need to create the same access right in that app, which duplicated the same thing in purchase app.

Anyway, I will add a test case in purchase app later.